app/api/v1/endpoints/admin.py

```
@@ -0,0 +1,237 @@
+
+        # Get timezone from settings
+        timezone = user_service.get_user_timezone(user.id)
+
+        # Build response
+        user_dict = user.model_dump()
+        user_dict['time_zone'] = timezone
+        user_dict['is_oidc_user'] = False  # Admin-created users are local users
+
+        return UserResponse.model_validate(user_dict)
+    except ValueError as e:
+        raise HTTPException(status_code=400, detail=str(e))
```

Ruff (0.14.5)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)